sdk/servicebus/azure-servicebus/azure/servicebus/management/_management_client.py

```diff
@@ -0,0 +1,287 @@
+        # type: (int, int, Type[Model]) -> Union[List[QueueDescription], List[QueueRuntimeInfo]]
+        with _handle_response_error():
+            et = cast(
+                ElementTree,
+                self._impl.list_entities(
+                    entity_type="queues", skip=skip, top=max_count, api_version=constants.API_VERSION
+                )
+            )
+        entries = et.findall(constants.ENTRY_TAG)
+        queues = []
+        for entry in entries:
+            entity_name = entry.find(constants.TITLE_TAG).text  # type: ignore
+            queue_description = _convert_xml_to_object(
+                entity_name,   # type: ignore
+                cast(Element, entry),
+                class_
+            )
+            queues.append(queue_description)
+        return queues
+
+    def get_queue(self, queue_name):
+        # type: (str) -> QueueDescription
+        """Get a QueueDescription
+
+        :param str queue_name: The name of the queue
+        """
+        return self._get_queue_object(queue_name, QueueDescription)
+
+    def get_queue_runtime_info(self, queue_name):
+        # type: (str) -> QueueRuntimeInfo
+        """Get the runtime information of a queue
+
+        :param str queue_name: The name of the queue
+        """
+        return self._get_queue_object(queue_name, QueueRuntimeInfo)
+
+    def create_queue(self, queue):
+        # type: (Union[str, QueueDescription]) -> QueueDescription
+        """Create a queue
+
+        :param queue: The queue name or a `QueueDescription` instance. When it's a str, it will be the name
+         of the created queue. Other properties of the created queue will have default values decided by the
+         ServiceBus. Use a `QueueDesceiption` if you want to set queue properties other than the queue name.
+        :type queue: Union[str, QueueDescription].
+        :returns: ~azure.servicebus.management.QueueDescription returned from ServiceBus.
+        """
+        try:
+            queue_name = queue.queue_name  # type: ignore
+            to_create = copy(queue)  # type: ignore
+            to_create.queue_name = None  # type: ignore
+        except AttributeError:
+            queue_name = queue  # type: ignore
+            to_create = QueueDescription()  # Use an empty queue description.
+
+        create_entity_body = CreateQueueBody(
+            content=CreateQueueBodyContent(
+                queue_description=to_create,  # type: ignore
+            )
+        )
+        request_body = create_entity_body.serialize(is_xml=True)
+        try:
+            with _handle_response_error():
+                et = cast(
+                    ElementTree,
+                    self._impl.queue.put(queue_name, request_body, api_version=constants.API_VERSION)  # type: ignore
+                )
+        except ValidationError:
+            # post-hoc try to give a somewhat-justifiable failure reason.
+            if isinstance(queue, (str, QueueDescription)):
```

Comment:
str->six.string_types? (I may have mangled the six.property name, but you get my drift :) )

Reply:
Changed sync code to this.